### PR TITLE
Fix cucumber tests and add eslint

### PR DESCRIPTION
**Pull request by stringhandler**
## Description

- Fix cucumber tests
- Add ESLint to integration tests
- Add ESLint to CI run

## Motivation and Context

To fix the tests.
Added ESLint to add standards for javascript

## Checklist:

* [x] I'm merging against the `development` branch.
* [] I have squashed my commits into a single commit.

### CODE REVIEW

**Comment by delta1** [comment]
Nice definitely needed

File: .circleci/config.yml [hunk 1]
```diff
@@ -92,6 +92,9 @@ commands:
      - run:
          name: Check formatting
          command: cd integration_tests && npm run check-fmt
+     - run:
+         name: Check eslint
+         command: cd integration_tests && npx eslint .
```
Comment (delta1) [inline .circleci/config.yml]: any reason to not add eslint to package.json with a script there?

File: integration_tests/cucumber.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
 module.exports = {
-  default: "",
-  critical: "--format @cucumber/pretty-formatter --tags @critical",
-};
+  default: '',
+  critical: '--format @cucumber/pretty-formatter --tags @critical'
+}
```
Comment (delta1) [inline integration_tests/cucumber.js]: I think there's a conflict between prettier and eslint here (single quotes vs double quotes, and semi-colons)

**Review by delta1 — COMMENTED** [review]
I think we just need to resolve differences between prettier and eslint

File: .circleci/config.yml (hunk 1 above)
Comment (stringhandler) [reply .circleci/config.yml]: lol, because my js is not great... that is the reason

**Pull request merged**